src/ajax/firebase-authentication.html

```diff
@@ -70,55 +70,81 @@
           type: Boolean,
           notify: true
         },
-        _tokens: Object
+        signedIn: Boolean,
+        _tokens: Object,
+        waitingForSignIn: Boolean
```

Comment:
Maybe you can also take advantage of [`statusKnown`](https://github.com/firebase/polymerfire/blob/master/firebase-auth.html#L115-L124)